packages/mobile/src/firebase/firebase.ts

```diff
@@ -103,14 +134,16 @@ const registerTokenToDb = async (app: Firebase, address: string, fcmToken: strin
   }
 }
 
-export const writePaymentRequest = (paymentInfo: PaymentRequest) => async () => {
+export function* writePaymentRequest(paymentInfo: PaymentRequest) {
   try {
     Logger.info(TAG, `Writing pending request to database`)
     const pendingRequestRef = firebase.database().ref(`pendingRequests`)
-    return pendingRequestRef.push(paymentInfo)
+    pendingRequestRef.push(paymentInfo)
```

Comment:
Does this need to be awaited or something? I'd guess this would create an unhandled promise here, surprised linter didn't catch it

Reply:
It returns a ThenableReference (https://firebase.google.com/docs/reference/js/firebase.database.ThenableReference.html), which still has .then and .catch methods - think you should still yield call this imo